Please fix the failing tests:

```
Executing pythonRemoveTestsDir
Finished executing pythonRemoveTestsDir
@nix { "action": "setPhase", "phase": "installCheckPhase" }
running install tests
...............................................................................E...............................
======================================================================
ERROR: Failure: PermissionError ([Errno 13] Permission denied: '/homeless-shelter')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/nix/store/1cb8wzfqlxv3xjbxcxsr6xd97yxixv59-python3.8-nose-1.3.7/lib/python3.8/site-packages/nose/failure.py", line 39, in runTest
    raise self.exc_val.with_traceback(self.tb)
  File "/nix/store/1cb8wzfqlxv3xjbxcxsr6xd97yxixv59-python3.8-nose-1.3.7/lib/python3.8/site-packages/nose/loader.py", line 416, in loadTestsFromName
    module = self.importer.importFromPath(
  File "/nix/store/1cb8wzfqlxv3xjbxcxsr6xd97yxixv59-python3.8-nose-1.3.7/lib/python3.8/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/nix/store/1cb8wzfqlxv3xjbxcxsr6xd97yxixv59-python3.8-nose-1.3.7/lib/python3.8/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/nix/store/1qkpav99nxjs14fswf78rhlq1xwhx5ch-python3-3.8.8/lib/python3.8/imp.py", line 234, in load_module
    return load_source(name, filename, file)
  File "/nix/store/1qkpav99nxjs14fswf78rhlq1xwhx5ch-python3-3.8.8/lib/python3.8/imp.py", line 171, in load_source
    module = _load(spec)
  File "<frozen importlib._bootstrap>", line 702, in _load
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/build/source/tests/core/test_versioning.py", line 21, in <module>
    from GTG.core import versioning
  File "/build/source/GTG/core/versioning.py", line 29, in <module>
    from GTG.core import datastore
  File "/build/source/GTG/core/datastore.py", line 43, in <module>
    class DataStore():
  File "/build/source/GTG/core/datastore.py", line 52, in DataStore
    def __init__(self, global_conf=CoreConfig()):
  File "/build/source/GTG/core/config.py", line 190, in __init__
    self._conf = open_config_file(self._conf_path)
  File "/build/source/GTG/core/config.py", line 79, in open_config_file
    os.makedirs(dirname)
  File "/nix/store/1qkpav99nxjs14fswf78rhlq1xwhx5ch-python3-3.8.8/lib/python3.8/os.py", line 213, in makedirs
    makedirs(head, exist_ok=exist_ok)
  File "/nix/store/1qkpav99nxjs14fswf78rhlq1xwhx5ch-python3-3.8.8/lib/python3.8/os.py", line 213, in makedirs
    makedirs(head, exist_ok=exist_ok)
  File "/nix/store/1qkpav99nxjs14fswf78rhlq1xwhx5ch-python3-3.8.8/lib/python3.8/os.py", line 223, in makedirs
    mkdir(name, mode)
PermissionError: [Errno 13] Permission denied: '/homeless-shelter'

----------------------------------------------------------------------
Ran 111 tests in 0.181s

FAILED (errors=1)
```

--- a/pkgs/applications/office/gtg/default.nix
+++ b/pkgs/applications/office/gtg/default.nix
@@ -16,13 +16,13 @@
 
 python3Packages.buildPythonApplication rec {
   pname = "gtg";
-  version = "unstable-2020-10-22";
+  version = "0.5";
 
   src = fetchFromGitHub {
     owner = "getting-things-gnome";
     repo = "gtg";
-    rev = "144814c16723fa9d00e17e047df5d79ab443fc5f";
-    sha256 = "1lpanfbj8y8b6cqp92lgbvfs8irrc5bsdffzcjcycazv19qm7z2n";
+    rev = "v${version}";
+    sha256 = "0b2slm7kjq6q8c7v4m7aqc8m1ynjxn3bl7445srpv1xc0dilq403";
   };
 
 
@@ -55,6 +55,10 @@ python3Packages.buildPythonApplication rec {
     mock
     xvfb_run
   ];
+
+  preBuild = ''
+    export HOME="$TMP"
+  '';
 
   format = "other";
   strictDeps = false; # gobject-introspection does not run with strictDeps (https://github.com/NixOS/nixpkgs/issues/56943)

